docs/refarch/boundedContexts/fspInteropApi/index.md

```diff
@@ -1,23 +1,248 @@
-# {name} BC
+# FSP Interoperability API BC
 
-{overview}
+The FSP Interoperability API Bounded Context provides access to the internal operations and resources that the Mojaloop Ecosystem provides to a given Participant. This Bounded Context is responsible for providing a Participant with interfaces whit which the Participant can perform and execute tasks on Mojaloop. It is also responsible for providing communication back to the Participant regarding different notifications and system messages that a participant should expect to receive.
+
+## Functional Overview
+
+The FSP IOP API interacts with many different bounded contexts and thus the simplified view is provided here, for further reading on the events and connections that the FSP IOP API provides and consumes please review the Mojaloop Common Interfaces [^1].
+
+![Use Case - Example REPLACE ME](./assets/0-functional-overview.jpg)
+
+> FSP Interoperability API Functional Overview
 
 ## Terms
 
 Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
 
-| Term | Description |
-|---|---|
-| Term1 | Description1 |
+| Term        | Description                                                                                                                                                      |
+| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Participant | Financial Service Provider that register on the Mojaloop ecosystem. Allowing said FSP to be able to transact with other Participants                             |
+| FSP IOP API | Financial Service Provider Interoperability Application Programming Interface, the API that provides access to functions and features of the Mojaloop Ecosystem. |
+| DFSP        | Digital Financial Service Provider                                                                                                                               |
 
 ## Use Cases
 
-### Perform Transfer (universal mode)
+Use case definitions [^2] have not been altered and the Reference Architecture allows for us to only change the background orchestration of the internal Mojaloop services and Bounded Contexts.
+
+### Account Lookup & Discovery BCs
+
+#### Party/Participant Associate
+
+Associate transfer/s Participant/s and/or parties based on the POST participant request [^5].
+
+![Use Case - Example REPLACE ME](./assets/0-1-party-participant-associate.jpg)
```

Comment:
Please change `ALT_TEXT_DESCRIPTION_GOES_HERE` the description text  as follows `![{ALT_TEXT_DESCRIPTION_GOES_HERE}](./assets/0-1-party-participant-associate.jpg)`.

This comment applies to the other images below.